static/sass/features/feature.scss

```diff
@@ -39,6 +39,18 @@ section {
   }
 }
 
+iron-icon {
```

Comment:
Can you add a comment here DRY this up with the rules in chromedash-feature.scss?